.github/workflows/lint.yml

```diff
@@ -50,3 +50,19 @@ jobs:
 
       - name: Run linter
         uses: ludeeus/action-shellcheck@2.0.0
+        with:
+          # Names of scripts with custom shebangs that the action would not
+          # discover on its own:
+          additional_files: >-
+            check_certificate_renewal
+            cpcd-config-up
+            discovery
+            finish
+            lock-tables-for-backup
+            matter-server-discovery
+            reboot
+            run
+            shutdown
+            universal-silabs-flasher-up
+            unlock-tables-for-backup
+            vlc-discovery
```

Comment:
It doesn't matter much, but for clarity, should we maybe use the full path for the add-on specific scripts, e.g. `vlc/rootfs/etc/s6-overlay/scripts/vlc-discovery`? :thinking: 

I'd still use just file name for the generic s6-init scripts (like `run` and friends).

Reply:
I would have loved to use full paths, but the `additional_files` input to the action only supports file names and not paths: https://github.com/ludeeus/action-shellcheck/blob/00b27aa7cb85167568cb48a3838b75f4265f2bca/action.yaml#L141-L143